lab-will/lib/basic-auth-middleware.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const createError = require('http-errors');
+const debug = require('debug')('cfgram:basic-auth-middleware');
```

Comment:
The name you are using here is cfgram.  But in your package.json you are telling debug to look for user.  This should be updated everywhere so it matches.  That way when you run your tests you can see all of your cool debug statements and you can narrow down where your code is breaking.